cmd/res/devices/intel.realsense.yaml.example

```
@@ -0,0 +1,9 @@
+deviceList:
+  - name: "Intel-Realsense"
+    profileName: "USB-Camera-General"
+    description: "Example Realsense Camera"
+    labels: [ "device-usb-camera-example", ]
+    protocols:
+      USB:
+        Paths: ["/dev/video0","/dev/video2","/dev/video4",]
```

Would it be better to split these out using the `-` syntax?

```suggestion
        Paths:
         - "/dev/video0"
         - "/dev/video2"
         - "/dev/video4"
```